serverless/knative_eventing/serverless-knative-eventing.adoc

```diff
@@ -12,12 +12,23 @@ Event-driven architecture is based on the concept of decoupled relationships bet
 
 For more information about event-driven architecture, see link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[What is event-driven architecture?]
 
-In a Knative Eventing workflow, event producers send information to an event source about changes to system state.
-Examples of event producers include a Kafka cluster or the Kubernetes API server.
-An event source is a resource object, which is the link between an event producer and a sink (or consumer) that receives those events.
-Examples of sinks are Knative services and channels.
-Events can also be sent to a broker, where they can be filtered using triggers before being sent to an event sink.
+== Knative Eventing workflows
 
-Using the broker and trigger together enables an event delivery mechanism that hides the details of event routing from the event producer and event consumer.
+In a Knative Eventing workflow, _event producers_ send information to an event source about changes to system state. Examples of event producers include a Kafka cluster or the Kubernetes API server.
 
-include::modules/serverless-eventing-components.adoc[leveloffset=+1]
+An _event source_ is a resource object, which is the link between an event producer and a _sink_ (or consumer) that receives those events. Currently, {ServerlessProductName} supports the following event source types:
+
+`ApiServerSource`:: Connects a sink to the Kubernetes API server.
+`PingSource`:: Periodically sends ping events with a constant payload. It can be used as a timer.
+`SinkBinding`:: Connects core Kubernetes resources like `Deployment`, `Job`, or `StatefulSet` with a sink.
```

Comment:
I don't think a sinkbinding belongs in this group.  It's not so much of a _source_ as it is the mechanism to track and bind the source and the sink.

Reply:
I'll take a look at this under another PR if that's OK

Reply:
Removed the mention of sinkbinding here, but I'll follow up with a review of the docs for this overall later to make sure it makes sense. Will add a new PR for any updates.